legacy/src/main/java/org/opensearch/sql/legacy/plugin/RestSQLQueryAction.java

```diff
@@ -149,7 +149,7 @@ public void onResponse(ExplainResponse response) {
           protected Object buildJsonObject(ExplainResponse response) {
             return response;
           }
-        }.format(response));
+        }.format(response), "application/json; charset=UTF-8");
```

Comment:
can the `getFormat()` from `protocol/src/main/java/org/opensearch/sql/protocol/response/format/JsonResponseFormatter.java` be used instead of the same string as it's public.

Reply:
Explain requests always use the `JsonResponseFormatter` class.  What we should do is save the formatter to a variable and reuse it to return the contentType.  Example: 
```
        JsonResponseFormatter formatter = new JsonResponseFormatter<ExplainResponse>(PRETTY) {
          @Override
          protected Object buildJsonObject(ExplainResponse response) {
            return response;
          }
        };
        sendResponse(channel, OK, formatter.format(response));
        }.format(response), formatter.getFormat());

```

Reply:
updated in https://github.com/Bit-Quill/opensearch-project-sql/pull/279/commits/0df2a1b34b182cea23bfb490d7143bd726aa00d4